### PR TITLE
Adding document group for AWS tools

### PR DESCRIPTION
**Pull request by jdmooney**

### CODE REVIEW

File: userguide/tutorials/errors-rollbar.adoc [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+First, go to the https://rollbar.com/[Rollbar website]. Rollbar
+offers a 30-day free trial, then continues free as long as your usage does not exceed certain limits. Provide a name and email address to sign up.
+Once you create an account, you should see the Rollbar main page.
+
+Kill Bill is preconfigured to send its error logs to Rollbar. No additional agent is needed. All that has to be done is to enable the transfer by providing an access token. The access token is installed by editing a simple configuration file.  
+
+== Get the Access Token
+
+To get the access token you will need, follow these steps:
+
```
Comment (reshmabidikar) [inline userguide/tutorials/errors-rollbar.adoc:30]: Minor correction,  `To get the access token you will need to`

File: userguide/tutorials/metrics-datadog.adoc [hunk 2]
```diff
@@ -3,60 +3,28 @@
...
-can be configured to provide its full collection of Kill Bill metrics to
-Datadog (and several other analysis tools). If you open the Kill Bill
-admin page in your browser, it should show that you are
-running the "Kill Bill Premium edition":
-
-image::metrics-kb-admin.png[align=center]
+Datadog can be integrated with several types of Kill Bill installations. In this document we discuss the use of Datadog to manage metrics produced
+by single-tier and multi-tier implementations on Amazon Web Services (AWS).
+With the latest Amazon Machine Image (AMI)
+for Kill Bill, the application is preconfigured to collect these metrics and to direct them to Datadog once authorization is provided.
```
Comment (reshmabidikar) [inline userguide/tutorials/metrics-datadog.adoc:19]: All the other docs specify that ` These instructions apply to the Kill Bill AMI version 2021.9 or later.`, so maybe we should add it to this doc as well.

File: userguide/tutorials/metrics-datadog.adoc [hunk 3]
```diff
@@ -3,60 +3,28 @@
...
+edit a configuration file to enable the transfer of the complete Kill Bill metrics.
 
-If you see this page you are ready to go. If not, you should upgrade
-your installation.
-
-While you are on this page, if you want to see all the metrics that Kill
-Bill has to offer, just add the path `/1.0/metrics` to the URL. Have fun!
-If you find these hard to understand, though, maybe datadog can help.
-
+
```
Comment (reshmabidikar) [inline userguide/tutorials/metrics-datadog.adoc:26]: The previous version of this document doc includes this sentence: `While you are on this page if you want to see all the metrics that Kill Bill has to offer, just add the path /1.0/metrics to the URL.`  If this is important, we should add it to the new doc as well.

File: userguide/tutorials/metrics-datadog.adoc [hunk 4]
```diff
@@ -68,57 +36,17 @@ the page.
...
-
-To test the installation, return to datadog and choose *Infrastructure*,
-then *Infrastructure List* from the left menu. The table that appears
-should list all of your hosts and indicate their status as *ACTIVE*. You
-may need to wait a few minutes for the table to appear.
-
-Now select *Metrics*, then *Summary* from the menu. You should see a
-long list of metrics! These are coming from your instances, but they
-only describe the Ubuntu environment itself. You do not yet have the
-Kill Bill metrics. To fix this, proceed to the next step.
```
Comment (reshmabidikar) [inline userguide/tutorials/metrics-datadog.adoc:41]: Is this section intentionally deleted from the new doc?

File: userguide/tutorials/metrics-newrelic.adoc [hunk 5]
```diff
@@ -3,56 +3,23 @@
...
 
-If you see this page you are ready to go. If not, you should upgrade
-your installation.
-
-While you are on this page, if you want to see all the metrics that Kill
-Bill has to offer, just add the path `/1.0/metrics` to the URL. Have fun!
-If you find these hard to understand, though, maybe New Relic can help.
+To set up New Relic to manage the metrics available from Kill Bill, there
+are just two steps: (1) setup a New Relic account, and (2)
+edit a configuration file to enable the transfer of the complete Kill Bill metrics.
```
Comment (reshmabidikar) [inline userguide/tutorials/metrics-newrelic.adoc]: Maybe split the steps into two separate lines to improve readability

File: userguide/tutorials/metrics-datadog.adoc [hunk 6]
```diff
@@ -3,60 +3,28 @@
...
-
-image::metrics-kb-admin.png[align=center]
+Datadog can be integrated with several types of Kill Bill installations. In this document we discuss the use of Datadog to manage metrics produced
+by single-tier and multi-tier implementations on Amazon Web Services (AWS).
+With the latest Amazon Machine Image (AMI)
+for Kill Bill, the application is preconfigured to collect these metrics and to direct them to Datadog once authorization is provided.
 
+To set up Datadog to manage the metrics available from Kill Bill, there
+are just two steps: (1) setup a Datadog account, and (2)
+edit a configuration file to enable the transfer of the complete Kill Bill metrics.
```
Comment (reshmabidikar) [inline userguide/tutorials/metrics-datadog.adoc:25]: Maybe split the steps into two separate lines to improve readability

**Review by reshmabidikar — COMMENTED** [review]
Looks good overall, Left some minor comments.

File: userguide/tutorials/errors-rollbar.adoc (hunk 1 above)
Comment (jdmooney) [reply userguide/tutorials/errors-rollbar.adoc:30]: I think the difference is I intended an imperative sentence (a direct instruction). Is the other form preferred?

File: userguide/tutorials/metrics-datadog.adoc (hunk 2 above)
Comment (jdmooney) [reply userguide/tutorials/metrics-datadog.adoc:19]: Fixed

File: userguide/tutorials/metrics-datadog.adoc (hunk 3 above)
Comment (jdmooney) [reply userguide/tutorials/metrics-datadog.adoc:26]: The page in question was moved to  aws-tools.adoc. The comment seemed less appropriate there.

File: userguide/tutorials/metrics-datadog.adoc (hunk 4 above)
Comment (jdmooney) [reply userguide/tutorials/metrics-datadog.adoc:41]: Yes. All the metrics appear at the same time now.

File: userguide/tutorials/metrics-datadog.adoc (hunk 6 above)
Comment (jdmooney) [reply userguide/tutorials/metrics-datadog.adoc:25]: Done.

File: userguide/tutorials/metrics-newrelic.adoc (hunk 5 above)
Comment (jdmooney) [reply userguide/tutorials/metrics-newrelic.adoc]: Done. I was debating whether this should be a list.

**Comment by jdmooney** [comment]
Updated in response to @reshmabidikar 's comments

**Comment by sbrossie** [comment]
@jdmooney Looks like there is a conflict that needs to be resolved

File: userguide/tutorials/errors-rollbar.adoc (hunk 1 above)
Comment (reshmabidikar) [reply userguide/tutorials/errors-rollbar.adoc:30]: Ahh ok, looks good in that case, I misunderstood 👍

**Review by reshmabidikar — APPROVED** [review]

**Comment by jdmooney** [comment]
Am I the one who should resolve the conflict? If so, how? We need to delete the existing file entirely and replace it with the new one.

**Comment by sbrossie** [comment]
> Am I the one who should resolve the conflict? 

Typically whoever generates a PR is on the hook to fix the conflict, so I would say 'yes'. 

> If so, how? We need to delete the existing file entirely and replace it with the new one.

If this is what needs to be done, then do it. You are closer to what these docs contain than anybody else.

**Pull request merged**